go/vt/vtctl/grpcvtctldserver/server.go

```diff
@@ -2461,6 +2461,15 @@ func (s *VtctldServer) GetUnresolvedTransactions(ctx context.Context, req *vtctl
 			if err != nil {
 				return err
 			}
+			// The metadata manager is itself not part of the list of participants.
+			// We should it to the list before showing it to the users.
```

Comment:
Will the metadata manager show up multiple times in the final list if we add it to every shardTrnx?

Reply:
Every Transaction will have it's own participant list, so metdata manager will get added to each transaction participant list once.

Reply:
Harshit is correct. We're adding it once to each transaction.